code/modules/mob/living/simple_animal/borer/borer_powers.dm

```diff
@@ -9,13 +9,12 @@
 
 	if(stat)
 		src << "You cannot leave your host in your current state."
+		return
 
 	if(docile)
 		src << "<span class='notice'>You are feeling far too docile to do that.</span>"
 		return
 
-	if(!host || !src) return
```

Comment:
This was already checked for above. Not sure how it would manage to happen between then and now.